packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesModule.kt

```diff
@@ -59,6 +62,7 @@ class UpdatesModule : Module(), IUpdatesEventManagerObserver {
 
     OnDestroy {
       UpdatesController.removeUpdatesEventManagerObserver()
+      ReloadScreenManager.unregisterReloadListener()
```

Comment:
can we unregister the listener earlier once we dismiss the reload view? or maybe register before `reloadAsync` call and unregister after the view dismissed